src/WPML_OptionsManager.php

```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
+            <?php $this->displayMP3Banner() ?>
         </div>
         <?php
     }
 
+    private function displayMP3Banner() {
+        ?>
+        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;padding: 20px;position: relative;">
+            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
+            <?php
+              if (rand(1, 2) === 1) {
+                    $this->displayMP3BannerVersionA();
+              } else {
+                    $this->displayMP3BannerVersionB();
+              }
+            ?>
+        </div>
+        <?php
+    }
+
+    private function displayMP3BannerVersionA() {
+        ?>
+            <ul style="list-style-type:disc">
```

Comment:
Can we align the list bullets with the title?
![image](https://user-images.githubusercontent.com/9365088/66461453-ba5f4f80-ea81-11e9-9678-4622cb2ef00d.png)

Reply:
I don't know, this is the default...

Reply:
Please indent. Looks ugly like this indeed. I'll let you pick a good value.